xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -188,51 +181,66 @@ public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
                       .withDescription("No traffic director provided by bootstrap")));
           return;
         }
-        // Currently we only support using the first server from bootstrap.
-        ServerInfo serverInfo = serverList.get(0);
-        channel = initLbChannel(
-            lookasideLbHelper, serverInfo.getServerUri(),
-            serverInfo.getChannelCredentials());
-        xdsClientRef = new RefCountedXdsClientObjectPool(new XdsClientFactory() {
+        XdsClientFactory xdsClientFactory = new XdsClientFactory() {
```

Comment:
It's either to make EDS-only case overkill or to make non-EDS/common(i.e. shutdown) case more complex. I took the trade-off to let non-EDS and common case simple, since EDS-only is also kind of legacy case.